test/integration/iam-policy-management.v1.test.js

```diff
@@ -801,6 +845,20 @@ describe('IamPolicyManagementV1_integration', () => {
       testTemplateId = result.id;
       testTemplateVersion = result.version;
     });
+    test('Create a S2S policy template', async () => {
+      const response = await service.createPolicyTemplate({
+        ...testS2STemplate,
+        accountId: testAccountId,
+      });
+      expect(response).toBeDefined();
+      expect(response.status).toEqual(201);
+      const { result } = response || {};
+      expect(result).toBeDefined();
+      expect(result.policy).toEqual(testS2STemplate.policy);
+      expect(result.state).toEqual('active');
+      testS2STemplateId = result.id;
```

Comment:
You can automate the generation of "link variables" that are used to implement dependencies between operations (e.g. the "id" response property here) by adding link objects to your API definition.
Here's a video tutorial on how to do this: https://secure.video.ibm.com/channel/23887899/playlist/651457/video/133486086